packaging/windows/Jenkinsfile.groovy

```diff
@@ -0,0 +1,72 @@
+def getCommit(path) {
+    dir(path) {
+        return bat(returnStdout: true, script: '@echo off && git rev-parse HEAD').trim()
+    }
+}
+
+def doBuild(){
+  stage 'Checkout Client'
+  bat 'if EXIST src\\github.com\\keybase\\client\\shared cd src\\github.com\\keybase\\client && git checkout shared'
+  bat 'if EXIST src\\github.com\\keybase\\client\\desktop\\shared cd src\\github.com\\keybase\\client && git checkout desktop/shared'
+  checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '${ClientRevision}']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'src\\github.com\\keybase\\client']], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/keybase/client.git']]]
```

Comment:
Please use whitespace and parens when there is more than one parameter to a function.
Also, the `RelativeTargetDirectory` stuff is pretty hard to read. IMO `dir` would be better.
```groovy
dir('src/github.com/keybase/client') {
    checkout([
        poll: false,
        scm: [
            $class: 'GitSCM',
            branches: [[name: '${ClientRevision}']],
            userRemoteConfigs: [[url: 'https://github.com/keybase/client.git']]
        ],
    ])
}
```